src/components/__test__/PhoneBrands.test.js

```diff
@@ -0,0 +1,19 @@
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import PhoneBrands from '../PhoneBrands';
+import App from '../../App';
+import store from '../../redux/store';
+
+describe(' Phone Brands Component Rendering ', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <App>
+            <PhoneBrands />
+          </App>
+        </Provider>,
+      );
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] you should write more tests for most components Also you should write tests for some pure functions in Redux code.